Cookbook/Technical-Documents/WorkingAtBabylon.md

```diff
@@ -0,0 +1,96 @@
+<p align="center">
+<img src="logo.png">
+</p>
+
+
+Working in the iOS team
+==================================
+
+This document is modelled after Robin Malhotra's [Questions to ask your iOS interviewer](https://github.com/codeOfRobin/Questions-to-ask-your-interviewer-iOS). It aims to provide a few answers to how we work at Babylon and our process is structured.
+
+### Where do we store our code?
+
+We use Github. We have a central repo that hosts the main and white label apps, and a [few Open Source projects](https://github.com/search?q=topic%3Aios+org%3ABabylonpartners+fork%3Atrue) that we created and rely on.
+
+## Continuous Integration
+### What Continuous Integration service do you use?
+
+We currently run on CircleCI
+
+### How many apps/targets/schemes?
+
+We have the main app and a few white-label apps.
+
+We use a modular architecture, so our app workspace is split into 9 Xcode projects for modules and apps, 3 projects for test utilities and shared features, and 4 projects related to our SDK, not counting the Pods project (and our OSS projects)
+
+Each module project has 2 targets on average: one for the framework, one for the tests (Unit, UI, Snapshot). The app project has 2 targets (6*2) given our white-labelling variants.
+### How big is your Fastfile?
+It's around 250 lines of code at the moment, with about 15-20 lanes for beta testing, daily and custom builds, etc. This file however includes multiple other files, especially in fastlane/Lanes and fastlane/Actions too, making the code dedicated to fastlane quite bigger.
+
+### Can you dry run a lane on your machine right now?
+Yes.
+
+* What architectural patterns are you using to make development easier or more consistent? What problems with your current approach have you come across after settling on one (if you have one)?
```

Comment:
that point should be either described or removed

Reply:
I've removed it for now.